Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -0,0 +1,106 @@
+import Library
+import Prelude
+import UIKit
+
+final class PledgeShippingLocationCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var adaptableStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var amountLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var countryButton: UIButton = { UIButton(frame: .zero) }()
+  private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var spacer: UIView = {
+    UIView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    _ = self
+      |> \.accessibilityElements .~ [self.titleLabel, self.countryButton, self.amountLabel]
+
+    _ = (self.rootStackView, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    self.rootStackView.addArrangedSubview(self.titleLabel)
+    self.rootStackView.addArrangedSubview(self.adaptableStackView)
+
+    self.adaptableStackView.addArrangedSubview(self.countryButton)
+    self.adaptableStackView.addArrangedSubview(self.spacer)
+    self.adaptableStackView.addArrangedSubview(self.amountLabel)
```

Comment:
Wondering if we can do something like this with stack views:

```
    _ = ([self.countryButton, self.spacer, self.amountLabel], self.adaptableStackView)
      |> ksr_addArrangedSubviews()
```

By defining:

```
public func ksr_addArrangedSubviews() -> (([UIView], UIStackView) -> UIStackView) {
  return { (subviews, stackView) in
    subviews.forEach { subview in
      stackView.addArrangedSubview(subview)
    }

    return stackView
  }
}
```

Reply:
I like this! ❤️

Reply:
I've slightly tweaked the naming so that it more resembles our other functions on `UIView`

i.e. just like when you're reading this from left to right

```
_ = (subview, parent)
  |> ksr_addSubviewToParent() // subview to parent (same order)
  |> ksr_constrainViewToEdgesInParent() // view to parent (same order)
```

we now do

```
_ = ([firstView, secondView, thirdView], stackView)
      |> ksr_addArrangedSubviewsToStackView()
```